app/views/posts/index.html.erb

```diff
@@ -1 +1,40 @@
-<h1>User 1 all posts</h1>
+<div class="container">
+  <!-- User Information Section -->
+  <div class="user_info">
+    <div class="img">
+      <!-- Display user photo using Rails image_tag helper -->
+      <%= image_tag @user.photo, alt: "User Photo", class: "user-photo" %>
+    </div>
+    <div class="user">
+      <!-- Display user name with a link to user profile -->
+      <h2><%= link_to @user.name, user_path(@user) %></h2>
+      <!-- Display the number of posts by the user -->
+      <p><strong>Number of Posts:</strong> <%= @user.posts.count %></p>
+    </div>
+  </div>
+
+  <!-- Display the two most recent posts by the user -->
+  <% @posts.limit(2).each do |post| %>
```

Comment:
- [ ] Please remove this `.limits(2)` because in the `posts index` page you will show all your posts so kindly remove this :slightly_smiling_face: